.github/workflows/dependency-review.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@4.0.0
+      - uses: actions/setup-go@v5
```

Comment:
`setup-go` version is the culprit? 🤔

Reply:
no, version is a small part. 
The real problem is the `securego/gosec@master` does not support `working-directory` and it cannot load our dependencies, [example failures](https://github.com/awslabs/amazon-eks-ami/actions/runs/7729953567/job/21074270364) and the cli way works

Reply:
oh got it, got confused based on just looking at `commit <-> checks`